mz_bokeh_package/environment.py

```diff
@@ -30,13 +30,15 @@ def get_request_url(endpoint: str) -> str:
     env = get_environment()
 
     if env == 'staging':
-        return 'staging.materials.zone:5000'
+        host = 'staging.materials.zone:5000'
     elif env == 'production':
-        return 'production.materials.zone:5000'
+        host = 'production.materials.zone:5000'
     elif env == 'dev':
-        return os.getenv('API_HOST', 'staging.materials.zone:5000')
+        host = os.getenv('API_HOST', 'staging.materials.zone:5000')
     else:
-        return "error"
+        host = "error"
```

Comment:
@roiweinreb , here "error" doesn't make much sense as a host. I think it's better to raise an ValueError exception in this case (and then I would change it everywhere). Anyways, this case should only happen if the user has a typo in writing staging, production or dev.